### DISCUSSION TITLE
Disable github activity shown in the feed without turning profile private

### DISCUSSION THREAD

**Discussion by prathikshetty2002** [community/community#20454] [Other Feature Feedback, Questions, & Ideas] [answered]
It is good to know what others are upto, but the feature leads to some privacy concerns. some people might not want to show their activity publically. so better give some option like turn off github activity. so from now onwards that person will not recieve that notifications of other people as well his/her activity will not be notified to others as well. This might be a good measure to implement.

**Comment by jfullstackdev** [comment, marked as answer]
> so better give some option like turn off github activity. so from now onwards that person will not recieve that notifications

what do you mean by that ? no one is being notified by that, if someone is following you, then it will appear on their feed, if someone watches a repo, they will be notified

**Reply by prathikshetty2002** [reply]
> > so better give some option like turn off github activity. so from now onwards that person will not recieve that notifications
> 
> what do you mean by that ? no one is being notified by that, if someone is following you, then it will appear on their feed, if someone watches a repo, they will be notified

yeah right but their is a new feature that allows to disable your activity. your followers can't see your activity , it kinda good for people who want privacy and don't want to show their activity. But for a person like me i want to showcase my activity but don't want to turn my profile private instead hide specific activities like following/stars etc. while keep certain activity. Github might bring this feature too. 

here is the option to turn off your activity:

![Screenshot 2022-07-14 235842](https://user-images.githubusercontent.com/63772910/179056085-ad6da98e-a67b-4569-bbae-12c2db72d70d.png)

**Reply by jfullstackdev** [reply]
yup, I know that, it was the answer to make a profile private, 

being notified by your timeline activity, as you mentioned, is not happening, the feed and the notifications are two different things

so by enabling it, you make your profile private and you are sure with that, it's just like turning off online status in Messenger, to be fair, you will not be seeing others that are active too, it's both ways, in our case here, it's all or nothing 

and hopefully, as your request, they may implement that too in the future but with no assurance

**Reply by prathikshetty2002** [reply]
> yup, I know that, it was the answer to make a profile private,
> 
> being notified by your timeline activity, as you mentioned, is not happening, the feed and the notifications are two different things
> 
> so by enabling it, you make your profile private and you are sure with that, it's just like turning off online status in Messenger, to be fair, you will not be seeing others that are active too, it's both ways, in our case here, it's all or nothing
> 
> and hopefully, as your request, they may implement that too in the future but with no assurance

yeah right. They are making a lot of changes lately, today i noticed the color scheme also changed from green to blue.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
